README.md

```diff
@@ -6,12 +6,12 @@ Libpod provides a library for applications looking to use the Container Pod conc
 popularized by Kubernetes.  Libpod also contains the Pod Manager tool `(Podman)`. Podman manages pods, containers, container images, and container volumes.
 
 * [Latest Version: 1.9.2](https://github.com/containers/libpod/releases/latest)
+  * Latest Remote client for Windows
+  * Latest Remote client for MacOs
+  * Latest Static Remote client for Linux
+
```

Comment:
Maybe I'm missing some GitHub magic, but how does this fix #6033?  This is just adding some bulleted text, not links to the distros no?

Reply:
Fixes it by removing the broken links.  #6033 also needs a fix from @mheon to ship the MacOS version in the release.